examples/forward/example-forward-single-variable-function-custom-scalar.cpp

```diff
@@ -0,0 +1,33 @@
+// C++ includes
+#include <iostream>
+#include <complex>
+using namespace std;
+
+// autodiff include
+#include <autodiff/forward.hpp>
+using namespace autodiff;
+
+// Specialize isNumber for complex to make it compatible with dual
+namespace autodiff::forward::traits {
+template<typename T>
+struct isNumber<complex<T>> : std::true_type { };
+} // autodiff::forward::traits
+
+using dual_cmpx = forward::Dual<complex<double>, complex<double>>;
```

Comment:
`cxdual` instead of `dual_cmpx`? :)

Reply:
cxdual is perfect)
I’ll change the name.